scripts/page_table.js

```diff
@@ -0,0 +1,36 @@
+window.addEventListener("DOMContentLoaded", () => {
+
+    let optionSelector = document.getElementById('option_selector');
+
+    function changeSelection(option) {
+        switch(option) {
+        case "all":
```

Comment:
Not sure if what I came up is cleaner, but just sharing it in case you did:

```
window.addEventListener("DOMContentLoaded", () => {

    let optionSelector = document.getElementById('option_selector');

    function changeSelection(option) {
      if (option === 'unproofed') {
        changeSelection('clear');
      }
      for (let box of document.getElementsByClassName('selectorbox')) {
        if (option === 'invert') {
          box.checked = option === 'all' || option === 'unproofed';
        } else if (option === 'all' || option === 'unproofed') {
          box.checked = true;
        } else {
          box.checked = false;
        }
      }
    };

    if(optionSelector) {
        optionSelector.addEventListener("change", function() {
            changeSelection(this.value);
        });
    }
});
```

Reply:
That is shorter, but maybe not so easy to understand (at least for me).

Reply:
Yeah I wasn't sure either. What you have is perfectly valid so I approved.